trusona.gemspec

```diff
@@ -28,6 +28,7 @@ Gem::Specification.new do |spec|
 
   spec.metadata['yard.run'] = 'yri' # use "yard" to build full HTML docs.
 
+  spec.add_development_dependency 'activesupport'
```

Comment:
`WARNING:  open-ended dependency on activesupport (>= 0, development) is not recommended
  use a bounded requirement, such as '~> x.y'`